cfbs.json

```diff
@@ -737,6 +737,35 @@
         "json cfbs/def.json def.json"
       ]
     },
+    "stig-canonical-ubuntu-20-lts": {
+      "description": "Canonical Ubuntu 20.04 LTS Security Technical Implementation Guide",
+      "tags": ["security", "compliance", "ubuntu"],
+      "repo": "https://github.com/nickanderson/cfengine-security-hardening",
+      "by": "https://github.com/nickanderson",
+      "version": "0.0.1",
+      "commit": "bdc7a16c6e52ff44149eb2abfd510fcee12dd2c1",
+      "subdirectory": "./stig/canonical_ubuntu_20.04_lts/",
+      "dependencies": [
+        "compliance-report-imports",
+        "library-parsed-local-groups"
+      ],
+      "steps": [
+        "copy ./config.cf services/security-hardening/stig/ubuntu-20/config.cf",
+        "copy ./V-238206/V-238206.cf services/security-hardening/stig/ubuntu-20/V-238206.cf",
+        "copy ./V-238204/V-238204.cf services/security-hardening/stig/ubuntu-20/V-238204.cf",
+        "copy ./V-238206/V-238206.cf services/security-hardening/stig/ubuntu-20/V-238206.cf",
+        "copy ./V-238201/V-238201.cf services/security-hardening/stig/ubuntu-20/V-238201.cf",
+        "copy ./V-238363/V-238363.cf services/security-hardening/stig/ubuntu-20/V-238363.cf",
+        "copy ./V-238219/V-238219.cf services/security-hardening/stig/ubuntu-20/V-238219.cf",
+        "copy ./V-238218/V-238218.cf services/security-hardening/stig/ubuntu-20/V-238218.cf",
+        "copy ./V-238215/V-238215.cf services/security-hardening/stig/ubuntu-20/V-238215.cf",
+        "copy ./V-251503/V-251503.cf services/security-hardening/stig/ubuntu-20/V-251503.cf",
+        "copy ./V-251504/V-251504.cf services/security-hardening/stig/ubuntu-20/V-251504.cf",
```

Comment:
Thar be bugz here!
https://github.com/nickanderson/cfengine-security-hardening/tree/master/stig/canonical_ubuntu_20.04_lts/V-251504

> To verify that null passwords cannot be used, run the following command:\
> $ grep nullok /etc/pam.d/common-password
> If this produces any output, it may be possible to log on with accounts with empty passwords.
> If null passwords can be used, this is a finding.


But, in the implementation .... currently, it says finding unless regline ...
https://github.com/nickanderson/cfengine-security-hardening/blob/bdc7a16c6e52ff44149eb2abfd510fcee12dd2c1/stig/canonical_ubuntu_20.04_lts/V-251504/V-251504.cf#L22

It should be if regeline().